.github/workflows/sourcemaps.yml

```
@@ -0,0 +1,46 @@
+          cache: 'pnpm'
+          cache-dependency-path: pnpm-lock.yaml
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build with sourcemaps
+        run: pnpm build
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+          NODE_ENV: production
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)